client/components/tinymce/plugins/simple-payments/dialog/list-item.jsx

```diff
@@ -40,6 +40,9 @@ class ProductListItem extends Component {
 	handleTrashClick = () => this.props.onTrashClick( this.props.paymentId );
 
 	formatPrice( price, currency = 'USD' ) {
+		if ( isNaN( price ) ) {
```

Comment:
Hmm, can we also add a stat or tracks event here? Maybe [logging to kibana](https://github.com/Automattic/wp-calypso/tree/master/client/lib/catch-js-errors#example) would be a better fit?